README.md

```diff
@@ -49,5 +49,5 @@ This is made specifically for Xbox console and hopefully, should feel familiar t
 	- Add-AppDevPackage.ps1
 	- Install.ps1
 
-Feel free to fork the repository and further develop the app to your liking, but you must keep branding & give credit where due.
+Feel free to make PR`s or even fork the repository and develop the app to your liking, but please keep branding, give credit where it`s due & keep in line with the licence. 
```

Comment:
Should say the exact same license as this source must be used and retained in any new works.